ovn-tester/ovn_tester.py

```diff
@@ -135,7 +136,13 @@ def read_config(configuration):
         density_light_args = config.get('density_light', dict())
         density_light_cfg = DensityCfg(
             n_pods=density_light_args.get('n_pods', 2),
-            lb=density_light_args.get('lb', False)
+            n_vips=density_light_args.get('n_vips', 0)
```

Comment:
This should be `n_vips=0`, if it's density light we don't want to run density-heavy by accident in `run_test_density()`